tests/functional/odf-cli/test_debug_verbocity_of_ceph_component.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from ocs_ci.helpers.helpers import odf_cli_set_log_level, get_ceph_log_level
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.ocs.exceptions import CommandFailed
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
```

Comment:
IMO, since this is cluster level test and not specific to pv services, it should be part of brown squad

Reply:
Done